### ISSUE TITLE
file transfer issue

### ISSUE THREAD

**Issue by danrobi11** [tofudd/lanchat#7]
Thanks for new file transfer feature.

I cant transfer anything I always get same error output

```
File exchange error: File transfer cancelled by user
File exchange error: Attempted to divide by zero.  
```

Lanchat 2.6 (linux binary not the flatpak)
OS: MX 19.3 (patito feo) x86_64 based Debian GNU/Linux 10 (buster)
SysV init version: 2.93
Kernel: 4.19.0-9-amd64

**Comment by tofudd** [comment]
What's size of your file?

**Comment by danrobi11** [comment]
> What's size of your file?

I've tried 364b and 7.9k

**Comment by tofudd** [comment]
Can you share log files? They should be in `~/.Lanchat2` folder.

**Comment by danrobi11** [comment]
```
[07:32:39] Logging started
[07:33:34] Session for 192.168.0.100 created. Session ID: 5a501621-cfcc-4898-99c1-c5745de1ec52
[07:33:34] Session 5a501621-cfcc-4898-99c1-c5745de1ec52 connected
[07:33:35] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received Handshake
[07:33:35] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received KeyInfo
[07:33:35] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received NodesList
[07:38:51] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
[07:39:33] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FilePart
[07:39:33] System.DivideByZeroException: Attempted to divide by zero.
   at Lanchat.Core.FileTransfer.FileTransferRequest.get_Progress()
[07:39:33] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
[07:40:09] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
[07:40:22] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FilePart
[07:40:22] System.DivideByZeroException: Attempted to divide by zero.
   at Lanchat.Core.FileTransfer.FileTransferRequest.get_Progress()
[07:40:22] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
[07:40:33] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
[07:40:48] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FilePart
[07:40:48] System.DivideByZeroException: Attempted to divide by zero.
   at Lanchat.Core.FileTransfer.FileTransferRequest.get_Progress()
[07:40:48] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
[07:41:37] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
[07:41:50] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FilePart
[07:41:50] System.DivideByZeroException: Attempted to divide by zero.
   at Lanchat.Core.FileTransfer.FileTransferRequest.get_Progress()
[07:41:50] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
[07:42:46] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
[07:43:06] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FilePart
[07:43:06] System.DivideByZeroException: Attempted to divide by zero.
   at Lanchat.Core.FileTransfer.FileTransferRequest.get_Progress()
[07:43:06] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
[07:43:35] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
[07:43:42] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FilePart
[07:43:42] System.DivideByZeroException: Attempted to divide by zero.
   at Lanchat.Core.FileTransfer.FileTransferRequest.get_Progress()
[07:43:42] Node 5a501621-cfcc-4898-99c1-c5745de1ec52 received FileTransferControl
```

**Comment by tofudd** [comment]
Fixed in f10a380b43f67d6819201a58c90270d469487d85. I'll merge it in a while.

**Comment by danrobi11** [comment]
Just tested v2.6.1 Working as intended. Thank you!
